lib/app/key_customization/key_customization.dart

```diff
@@ -70,15 +66,11 @@ class KeyCustomizationManager {
       'display_name': customName?.isNotEmpty == true ? customName : null
     };
     _log.debug('Setting key customization for $serial: $properties');
-    final sha = _getSerialSha(serial);
-    _customizations[sha] = properties;
+    _customizations[serial] = properties;
   }
 
   Future<void> write() async {
     await _prefs.setString(_prefKeyCustomizations,
-        base64UrlEncode(utf8.encode(json.encode(_customizations))));
+        String.fromCharCodes(utf8.encode(json.encode(_customizations))));
```

Comment:
This can just be `json.encode(_customizations)`, no need to encode to UTF8 and then back again.

Reply:
fixed